scripts/mods/mod_reforged/dynamic_spawns/parties/noble_parties.nut

```diff
@@ -1,7 +1,7 @@
 local parties = [
 	{
 		ID = "Noble",
-		HardMin = 5,
+		HardMin = 10,
```

Comment:
These HardMin changes for nobles are rather significant, what's the reasoning?

Reply:
I would also like to know that.
Feels very much like fighting the symptoms rather than the cause. If you feel they are too weak during early game then we should rather give them more resources then

Reply:
I discussed this with Uber on voice chat. He explained that he doesn't want Nobles and Mercenaries to be easily "farmable" by players in the early game, so he wants to force them to have bigger parties earlier on - but doesn't want to change their strength in mid-late game from how it is right now. And this seems like an acceptable way to achieve that goal.